perfomance_tests/expression/expression_test.go

```diff
@@ -26,19 +31,52 @@ func BenchmarkDefault2Expr(b *testing.B) {
 		MainTargetValue: 10.0,
 		WarnValue:       &warnValue,
 		ErrorValue:      &errorValue,
+		TriggerType:     moira.RisingTrigger,
```

Comment:
hmmm, why it become necessary?

Reply:
I debugged the benchmark and it was actually returning errors, therefore the tests were not really testing the speed of expression parsing.

Reply:
:eyes:

Reply:
I think this issue can be resolved already. @kissken ?